bundle/src/main/java/com/adobe/acs/commons/indesign/dynamicdeckdynamo/models/CollectionSelectComponent.java

```diff
@@ -41,8 +41,11 @@
     @Override
     public Map<String, String> getOptions() {
         Map<String, String> options = new LinkedHashMap<>();
-        ResourceResolver resourceResolver = getHelper().getRequest().getResourceResolver();
-        if (null != resourceResolver) {
+        if (null != getHelper() &&
```

Comment:
`getHelper()` returned null in the "unrelated" `DialogResourceProviderFactoryTest`